recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ build:
 
 requirements:
   build:
+    - gnuconfig  # [unix]
```

Comment:
What is `gnuconfig`?

Reply:
It's the new package with `config.sub` and `config.guess`.

Reply:
Sorry so what does it do?